test/common/common/lambda_logger_delegate_test.cc

```diff
@@ -0,0 +1,38 @@
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+#include "library/common/common/lambda_logger_delegate.h"
+
+using testing::_;
+using testing::HasSubstr;
+
+namespace Envoy {
+namespace Logger {
+
+TEST(LambdaDelegate, LogCb) {
+  uint i{0};
+  std::string expected_msg = "Hello LambdaDelegate";
+
+  LambdaDelegate delegate = LambdaDelegate(
+      [&i, &expected_msg](absl::string_view msg) -> void {
+        EXPECT_THAT(msg, HasSubstr(expected_msg));
```

Comment:
This is a nit. I generally prefer the expectations to be at the bottom of the test so maybe instead of incrementing an uint, could we capture the value passed and assert this captured value instead of asserting the uint?

Reply:
I like that, updated.